scripts/suite.py

```diff
@@ -172,6 +172,8 @@
                               in the output of teuthology-suite command. -1
                               for a random seed [default: -1].
  --force-priority             Skip the priority check.
+ --ignore-job-limit           Skip the number of jobs check. Use this to schedule
+                              more than DEFAULT_JOB_LIMIT=500 jobs.
```

Comment:
Maybe rename to --ignore-job-limit, also it would be better to say couple of words what is the default limit and that it is used, here. Or even have --limit-job options defaults to 500, and we need to go with maximum available we can use parameter equals 0 or '-1'.
I agree this is a bit weird to discuss this a posteriori.

Reply:
ah... what do you think about using that `  -l <jobs>, --limit <jobs>   Queue at most this many jobs` option instead?

Reply:
but if we change the default job limit to 500 then functionality of sleep before teardown will be broken, the code following code should be fixed: 
https://github.com/ceph/teuthology/blob/master/teuthology/suite/run.py#L563-L573

Reply:
like:
```
if job_limit == 0 or job_limit == DEFAULT_JOB_LIMIT:
```

Reply:
also, if we use --limit options, we do not need to check jobs number to schedule because it will not be generated more than the default using existing code as far as I can see.

Reply:
This is different -- it is used as a cut off for the number of (often similar) jobs to be scheduled.  For example, if a `workload/foo` facet has 10 different variations and you only want to schedule one of them, `--filter workload/foo --limit 1` would do that.

Reply:
The --limit option is highly dangerous for a large suite like rados, and not encouraged at all.

Reply:
yeah, thanks, I agree it is a mess, the feature is in alerting not limiting